docs/wiki/configurator/failsafe-tab.md

```diff
@@ -4,8 +4,135 @@ sidebar_position: 5
 
 # Failsafe Tab
 
+Failsafe is the action your craft takes when it loses connection to the transmitter.
+This tab allows you to configure what your craft does when it loses connection to the transmitter.
```

Comment:
Mostly just gonna comment slight fixes here and there, starting with this:

"loses connection to the transmitter." is repeated twice, could be worded a little more cleanly

Reply:
Should I put:

Failsafe is the action your craft takes when it loses connection to the transmitter.
This tab allows you to configure what your craft does in an event of a failsafe.

Or something similar? What do you suggest?